### PR TITLE
Nerfs reactive armour

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Too much powergaming going on with it

### CODE REVIEW

**Comment by kevinz000** [comment]
NO

**Comment by newfren** [comment]
10/10

**Comment by kevinz000** [comment]
no metagrudging

**Comment by newfren** [comment]
it's the perfect nerf

**Comment by KazeEspada** [comment]
*dusts off the maintainer rulebook*  *puts on reading glasses* *scans through many pages* Ahh, here it is.
Rule: No CKEYs may be ever referenced directly in the code EXCEPT CKEYs that were already in the code before this rule. See: Rosham.

**Comment by PKPenguin321** [comment]
Ayy lmao

File: code/modules/clothing/suits/armor.dm [hunk 1]
```diff
@@ -157,6 +157,9 @@
 	..()
 
 /obj/item/clothing/suit/armor/reactive/hit_reaction(mob/living/carbon/human/owner, attack_text, final_block_chance)
+	if(owner.ckey == "kevinz000" || owner.real_name == "Mekhi Anderson")
+		owner.dust()
```
Comment (PKPenguin321) [inline code/modules/clothing/suits/armor.dm]: This is all fine and good but you forgot to add a flavor message to the user that insults the small size of his genitals

**Comment by kevinz000** [comment]
:^(

**Comment by PKPenguin321** [comment]
>inb4 this gets yolomerged

**Comment by KorPhaeron** [comment]
@PKPenguin321 

HG test merged it and Mekhi got dusted

**Comment by PKPenguin321** [comment]
If you're serious that's wonderful
Now that we know it works we just gotta wait 22 hours

**Comment by kevinz000** [comment]
pls no merge

**Comment by kevinz000** [comment]
no dorsisdwarf why?

**Comment by RemieRichards** [comment]
@KazeEspada Jokes on you that rule is one of the Unwritten ones, there's no book with it!

**Comment by Shadowlight213** [comment]
:+1: Next is flashbangs and flashes

**Comment by Durkel** [comment]
>you will never get a ingame or code reference.

Why live?

**Comment by GunHog** [comment]
I am getting sick of all these grudge code PRs, including the memes. One person (Mekhi) overuses something, and now some salty player with Github installed takes it away. It is horrible and it needs to stop.

**Comment by duncathan** [comment]
l m a o

**Comment by PKPenguin321** [comment]
reopen this to be very honest

**Pull request closed without merging**